src/style/helpers.scss

```diff
@@ -1,3 +1,52 @@
-.text-shadow-none{
+root {
+  --highlight-area-gray: #333333;
+}
+
+.text-shadow-none {
   text-shadow: none;
 }
+
+.center-items {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.login-page {
+  background-image: url("../assets/images/background/parallax-mountain.png");
+}
+
+.login-container {
+  opacity: .95;
+  background-color: $body-bg;
+  border-radius: 15px;
+
+  @media (min-width: 992px) {
+    width: 50%;
+    left: 25%;
+  }
+
+  @media (min-width: 1150px) {
+    width: 40%;
+    left: 30%;
+  }
+  @media (max-height: 659px) {
+    margin-top: 15px;
+  }
+  @media (min-height: 660px) {
+    margin-top: 10%;
+  }
+  padding-top: 15px
+}
+
+.login-intro .row {
+  justify-content: center;
+}
+
+.login-input .row:first-child {
+  justify-content: center;
+}
```

Comment:
you could also just use the class justify-content-center instead of writing custom css

Reply:
i think i had some problems using this class (could be because of the containers though). Will have another look at it